### PR TITLE
Update translations from Fedora Weblate

### PR DESCRIPTION
**Pull request by cockpituous**
Update translations from Fedora Weblate
 * [x] po-refresh

### CODE REVIEW

File: po/ja.po [hunk 1]
```diff
@@ -10,33 +10,29 @@ msgid ""
...
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=1; plural=0\n"
-"X-Generator: Weblate 3.10.3\n"
-
-#: pkg/docker/storage.jsx:261
-msgid " (shared with the OS)"
-msgstr " (OS と共有)"
+"Plural-Forms: nplurals=1; plural=0;\n"
```
Comment (marusak) [inline po/ja.po]: ech...
lets keep this PR open with `needs-work` and I'll take a look at this, as it is getting really annoying.

Comment (marusak) [reply po/ja.po]: One solution would be (likely) to use `sed -i "s/\(^.Plural-Forms:.*\);\\\n\"$/\1\\\n\"/" *.po` in `po-refresh`. Not ideal, but workaround.

I also want to take a look if we can use http://docs.translatehouse.org/projects/translate-toolkit/en/latest/commands/index.html

Comment (martinpitt) [reply po/ja.po]: Some seddery seems ok to me as a first bandaid.

**Comment by marusak** [comment]
This is obsolete, lets get a new run.

**Pull request closed without merging**